lib/metadataTypes/definitions/MobileKeyword.definition.js

```diff
@@ -16,14 +16,15 @@ module.exports = {
     typeName: 'Mobile Keyword',
     fields: {
         id: {
+            // irregular format: NXV4ZFMwTEFwRVczd3RaLUF5X3p5dzo4Njow
             isCreateable: false,
-            isUpdateable: false,
-            retrieving: false,
+            isUpdateable: true,
+            retrieving: true,
             template: false,
         },
         keyword: {
             isCreateable: true,
-            isUpdateable: false,
+            isUpdateable: true,
```

Comment:
is it updateable?

Reply:
it needed to be included during updates or else the API returned an error. no, you cannot change it.